openapi/p4pa-debt-position.openapi.yaml

```diff
@@ -0,0 +1,273 @@
+openapi: 3.0.1
+info:
+  title: p4pa-debt-positions
+  description: Api and Models
+  version: 0.0.1
+servers:
+  - url: http://localhost:8080
+    description: Generated server url
+paths:
+  /api/debt-positions:
+    post:
+      tags:
+        - debt-position-controller
+      operationId: createDebtPosition
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/DebtPositionDTO"
+        required: true
+      responses:
+        "201":
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DebtPositionDTO'
+        "400":
+          description: Invalid request
+        "500":
+          description: Internal server error
+components:
+  schemas:
+    DebtPositionDTO:
```

Comment:
can we identify also the required properties?
This DTO will be copy/pasted everywhere, even if it's not used on internal call, it could be reused for FE API.
Let's consider the table NOT NULL fields ;) (except for the id! they are filled by this application during save operation

Reply:
added required properties